tests/unit_tests/weightwindows/wwinp_cyl

```diff
@@ -0,0 +1,47 @@
+         1         1         2        16
+         0         4
+          6.0          8.0          1.0 -1.00000E+02 -1.00000E+02 -5.00000E+01
+          6.0          8.0          1.0          1.0
```

Comment:
For cylindrical/spherical, it looks like there's an extra six values (x1 y1 z2 x2 y2 z2) that you're missing in this example

Reply:
This is mainly to check that `wwinp_to_wws` is raising the an exception when a non-Cartesian mesh is passed in, but still, best to have valid files as test data. I'll update this to include the vectors and correct the 
phi mesh.